processes/third_party_vuln_process.md

```diff
@@ -92,4 +92,8 @@ as the [Node.js Security Team](https://github.com/nodejs/security-wg/blob/master
 Members of the security teams should indicate that they accept the privacy policies 
 by PRing their acceptance to this file:
```

Comment:
This suggests that people should raise individual PRs to add themselves to the list. I have no issue with us not doing that this time, but maybe @bengl @brycebaril @cjihrig and @sam-github should comment in this PR with their acceptance?

Reply:
@gibfahn you're actually right! My bad.

@bengl @brycebaril @cjihrig and @sam-github : I think the safest way is to close current PR and have you open new ones.

As no-one objected on this current one, I think we will be safe to merge your individual PRs immediately.

Sorry about this process mistake from me.

Reply:
I don't think I should be involved in the thirdparty triage at this time - I don't have the time. I do accept the privacy policies, though!